### ISSUE TITLE
Failing to build ‘grind’-component in RAP4

### ISSUE THREAD

**Issue by Svroozendaal** [AmpersandTarski/RAP#346]
#### De situatie
When trying to run the ATLAS component by using the Grind build recipe, an error occurs and the prototype fails to be build properly. 

![afbeelding](https://github.com/AmpersandTarski/RAP/assets/115573006/a48e0caf-59ad-4842-8961-34c785566756)

#### Wat er gebeurde er en wat had je verwacht?

An error occurs when running the RAP4 - Grind component instead of building the prototype

#### Stappen om dit te reproduceren

1. Clone the latest version of the RAP repository from Github
2. Navigate to  RAP4/Dockerfile, and alter the line 41 to:
`--build-recipe Grind\`

The RUN command should now look like this:
 
![afbeelding](https://github.com/AmpersandTarski/RAP/assets/115573006/a70f5d50-282a-4552-b996-fc7d18ad5d24)

Save the file

4.  Now try to generate the prototype by using the command 
`docker-compose up -d`

#### Screenshot / Video
![MicrosoftTeams-image](https://github.com/AmpersandTarski/RAP/assets/115573006/7c36423c-9a5e-4621-9e5d-70c494129156)

**Comment by hanjoosten** [comment]
Thanks for the report, @Svroozendaal :)

I have been able to reproduce this. I was curious why the change of the Recipe was done, so I also ran the docker-compose command without the change of the Dockerfile. As I expected, the application was deployed successfully. 

Depending on the purpose of what you are trying to achieve, I suggest not to change the recipe. This enables you to play around with Ampersand scripts and see what the Atlas does.
Maybe there was another purpose. In that case, we have to make clear what that purpose is. Only changing the recipe leads to this error, so it can never be sufficient to reach whatever purpose you have in mind. 
I expect the change you did has been suggested by @stefjoosten. Maybe he can comment on this too.
